python/scripts/hpx_run_test.py

```diff
@@ -217,18 +218,18 @@ def read_callback(fd, job):
     test_passed = True
 
     for result in results:
-      if not result[1]:
+      if (not result[1]):
         test_passed = False
         break
 
     all_passed = all_passed and test_passed
 
-    print "-", ("Passed" if test_passed else "Failed")
+    print ("-", ("Passed" if test_passed else "Failed"))
 
-    if "always" == options.log or ("fail" == options.log and not test_passed):
+    if ("always" == options.log or ("fail" == options.log and not test_passed)):
       f = None
 
-      if not options.log_stdout:
+      if (not options.log_stdout):
```

Comment:
I don't think the parenthesis are needed. They (arguably) make the code less readable.

Reply:
yes, it is removed